packages/@n8n/task-runner/src/config/base-runner-config.ts

```diff
@@ -14,7 +14,7 @@ class HealthcheckServerConfig {
 
 @Config
 export class BaseRunnerConfig {
-	@Env('N8N_RUNNERS_N8N_URI')
+	@Env('N8N_TASK_BROKER_URI')
```

Comment:
Should we rename also `n8nUri` for consistency?

Also the prefix is different now: `N8N_RUNNERS_` vs `N8N_TASK_BROKER`. Should we use the same everywhere?

Reply:
Good point! Updated the PRs to use `N8N_RUNNERS_` prefix for all